guides/command-implementation.md

```diff
@@ -26,3 +26,86 @@ have
    1. describe what this command is for (what it does)
    2. information that you think that users should be aware of
    3. sample(s) of the command
+
+## Guidelines
+
+1. Consistent verb and option naming conventions. For example, we standardize on
+   `-o, --org-name` where `-o` is the alias and `--org-name` is the full option
+   name for organization option. We do not use another aliases other than `-o`
+   and other name other name like `--organization-name`. See
+   [Option Name Section](#option-name)
+2. Consistent command syntax. We use `commander` open source library to enforce
```

Comment:
Can link to the repo for commander: https://github.com/tj/commander.js

Reply:
accept